src/arr/trove/error.arr

```diff
@@ -2526,15 +2554,15 @@ data ParseError:
             ED.text("Pyret thinks the string ")],
           ED.cmcode(self.loc),
           [ED.para:
-            ED.text("is unterminated; you may be missing closing punctuation. If you intended to write a multi-line string, use "),
+            ED.text("is not finished; you may be missing closing punctuation. If you intended to write a multi-line string, use "),
```

Comment:
Apologies; I didn't mean to include these two lines in this PR; they're unrelated.